docs/overview.md

```diff
@@ -13,22 +16,49 @@ As a versatile solution, ScalarDB supports a range of databases, including:
 
 For details on which databases ScalarDB supports, refer to [Supported Databases](scalardb-supported-databases.md).
 
+## Why ScalarDB?
+
+There are several solutions, such as global transaction managers, data federation engines, and HTAP systems, built towards similar goals, but they are limited in the following perspectives:
+
+- Global transaction managers (e.g., Oracle MicroTx, Atomikos) are designed to run transactions across a limited set of heterogeneous databases (e.g., only XA-compliant databases).
+- Data federation engines (e.g., Denodo, Starburst) are designed to run analytical queries across heterogeneous databases.
+- HTAP systems (e.g., TiDB, SingleStore) run both transactions and analytical queries only on homogeneous databases.
+
+In other words, they virtually unify databases in a half-baked way.
+For example, with data federation engines, users can run read-only analytical queries on a virtualized view across multiple databases. However, they often need to run update queries separately for each database.
```

Comment:
`half-baked` is a little too casual. How about `impractical` (as shown below), `uncomprehensive`, or `narrow`?

```suggestion
In other words, they virtually unify databases in an impractical way. For example, with data federation engines, users can run read-only analytical queries on a virtualized view across multiple databases. However, they often need to run update queries separately for each database.
```

Reply:
@josh-wong `incomplete` sounds more appropriate to me. How does it sound to you?

Reply:
`incomplete` works for me!